experimental/examples/imageviewer/shared/src/commonMain/kotlin/example/imageviewer/view/PreviewImage.common.kt

```diff
@@ -1,51 +1,85 @@
+import androidx.compose.animation.ExperimentalAnimationApi
+import androidx.compose.animation.slideInVertically
+import androidx.compose.animation.slideOutVertically
+import androidx.compose.animation.with
 import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
 import androidx.compose.foundation.clickable
-import androidx.compose.foundation.layout.*
-import androidx.compose.material.*
-import androidx.compose.runtime.*
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Card
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.ImageBitmap
-import androidx.compose.ui.graphics.RectangleShape
 import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.unit.dp
-import example.imageviewer.model.*
-import example.imageviewer.model.State
-import example.imageviewer.style.*
+import example.imageviewer.model.Picture
+import example.imageviewer.style.ImageviewerColors.kotlinHorizontalGradientBrush
 import org.jetbrains.compose.resources.ExperimentalResourceApi
-import org.jetbrains.compose.resources.orEmpty
-import org.jetbrains.compose.resources.rememberImageBitmap
-import org.jetbrains.compose.resources.resource
 
-@OptIn(ExperimentalResourceApi::class)
+@OptIn(ExperimentalResourceApi::class, ExperimentalAnimationApi::class)
 @Composable
-internal fun PreviewImage(state: MutableState<State>, getImage: suspend (Picture) -> ImageBitmap) {
-    val pictures = state.value.pictures
-    val index = state.value.currentImageIndex
-    val imageState = remember(pictures, index) { mutableStateOf<ImageBitmap?>(null) }
-    LaunchedEffect(pictures, index) {
-        val picture = pictures.getOrNull(index)
+internal fun PreviewImage(
+    picture: Picture?,
+    onClick: () -> Unit,
+    getImage: suspend (Picture) -> ImageBitmap
+) {
+    // TODO: Make this not dependent on galleryState
+    var image by remember(picture) { mutableStateOf<ImageBitmap?>(null) }
+    LaunchedEffect(picture) {
         if (picture != null) {
-            imageState.value = getImage(picture)
+            image = getImage(picture)
         }
     }
 
-    val image = imageState.value
+    Spacer(
+        modifier = Modifier.height(5.dp).fillMaxWidth()
+            .background(brush = kotlinHorizontalGradientBrush)
+    )
     Card(
-        backgroundColor = MaterialTheme.colors.background,
+//        colors = CardDefaults.cardColors()
```

Comment:
I do not see any reason why to leave commented out code. Here and below

Reply:
Yup, these were leftovers from the migration. I have removed the commented out code.